tutor/controllers/forms.py

```diff
@@ -19,4 +19,9 @@ class LoginForm(FlaskForm):
     password = PasswordField('Password',
                 validators=[DataRequired()])
     remember = BooleanField('Remeber Me')
-    submit = SubmitField('Login')
\ No newline at end of file
+    submit = SubmitField('Login')
+
```

Comment:
not sure, but aren't the development rules you've set for your project saying that each module gets its own file instead of collecting things in big central files that can have git conflicts?

Reply:
actually, I didn't work on it, just saw on some PR that a new line should have been added.. probably my bad fixing it.. @ifireball

Reply:
I don't understand. you're saying this code change in not related to this PR?

Reply:
yep.. was working on the file so also added a new line for another function that has nothing with the new features.
same as for another comment below

Reply:
I opened new issue for splitting forms.py. a split will happen on the next PR.